person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  def initialize(name, age, parent_permission)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id, :parent_permission
+  attr_accessor :name, :age
+
+  def of_age?
+    @age >= 18
+  end
```

Comment:
You have followed the requirements to create the of_age? method to check if the user is of age. Kindly make it a public method as specified in the [requirements](https://github.com/microverseinc/curriculum-ruby/blob/ee60e49e943e5e657dca23bc5b9145b771c42ca4/oop/school_library_people.md?plain=1#L30).

By default methods defined are public. To define a private method requires writing a private keyword after the public method(s) and before the private method(s). E.g

```
  def can_use_services? 
    is_of_age?() || @parent_permission
  end

  private 
 
  def is_of_age?
    @age >= 18
  end
```